independent-projects/bootstrap/runner/src/main/java/io/quarkus/bootstrap/forkjoin/QuarkusForkJoinWorkerThreadFactory.java

```diff
@@ -0,0 +1,16 @@
+package io.quarkus.bootstrap.forkjoin;
+
+import java.util.concurrent.ForkJoinPool;
+import java.util.concurrent.ForkJoinWorkerThread;
+
+public final class QuarkusForkJoinWorkerThreadFactory implements ForkJoinPool.ForkJoinWorkerThreadFactory {
```

Comment:
Maybe we want to apply this to `QuarkusDevModeTest`, `QuarkusUnitTest` and `QuarkusProdModeTest`?

Reply:
Probably? I'm not familiar with these

I do suspect thought that we'll want to have a slightly different implementation for "real prod mode" than for tests